.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build and verify
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [$default-branch]
```

Comment:
We saw in PicnicSupermarket/error-prone-support#278 that unfortunately this doesn't work.